test/Makefile

```diff
@@ -12,7 +12,7 @@ TEST_PAYLOAD := piggie/piggie
 TEST_BINARIES := test $(TEST_PAYLOAD) phaul/phaul
 COVERAGE_BINARIES := test.coverage phaul/phaul.coverage
 
-all: $(TEST_BINARIES) phaul-test
+all: $(TEST_BINARIES) phaul-test crit-test
```

Comment:
`crit-test` generates a significant amount of logs.
IMHO it would be better to run them in a separate step in [`.github/workflows/main.yml`](https://github.com/checkpoint-restore/go-criu/blob/a6d7cb17ecc29ac8f9adc19e18f8c5193a416f1a/.github/workflows/main.yml):

```
sudo make -C test/ crit-test
```